Implement Cardano transactions verification in `mithril-client-wasm` (#1512)
input-output-hk/mithril: mithril-build-script/README.md

# Mithril-build-script

> A toolbox of generations functions for Mithril crates that needs a [build scripts phase](https://doc.rust-lang.org/cargo/reference/build-scripts.html).
> 
> In addition this crates allow to tests those functions as tests are not supported directly in `build.rs` (see: [rust-lang/cargo#1581](https://github.com/rust-lang/cargo/issues/1581)).

Reviewer: ```suggestion
A toolbox for Mithril crates that need a [build scripts phase](https://doc.rust-lang.org/cargo/reference/build-scripts.html).

Moreover, this crate allows to test the build scripts, which is not supported directly in `build.rs` (see: [rust-lang/cargo#1581](https://github.com/rust-lang/cargo/issues/1581)).
```
